code/modules/research/techweb/all_nodes.dm

```diff
@@ -569,6 +569,7 @@
 		"pacman",
 		"plasma_tank",
 		"plasmaman_tank_belt",
+		"generic_tank",
```

Comment:
wtf is this lol

Reply:
this is the generic gas tank. this should be a separate pr